pkg/kubelet/dockershim/docker_container.go

```diff
@@ -243,10 +239,12 @@ func (ds *dockerService) createContainerLogSymlink(containerID string) error {
 			glog.V(10).Infof("Docker logging driver is %s", dockerLoggingDriver)
 		}
 
```

Comment:
(above, [line 236](https://github.com/kubernetes/kubernetes/pull/44126/files#diff-c7dd39479fd733354254e70845075db5R236)) Shouldn't that `glog.Errorf` call return an error instead? We don't want to carry on if the dockerInfo call didn't return anything useful...